README.md

```diff
@@ -108,7 +108,7 @@ lwc-jest --debug -- --no-cache
 If you prefer to debug inside Visual Studio Code, follow these steps:
 - From the Visual Studio Code dropdowns, select Debug > Add Configuration....
 - If you're prompted for an Environment choose any value.
-- Replace the contents of the generated `launch.json` with the following.
+- Replace the contents of the generated `launch.json` with the following (for Windows users see the [Jest website](https://jestjs.io/docs/en/troubleshooting#debugging-in-vs-code) for launch.json contents).
```

Comment:
Mac users, replace the contents of the generated `launch.json` with the following. (Windows users, see the [Jest website](https://jestjs.io/docs/en/troubleshooting#debugging-in-vs-code) for launch.json contents).